### PR TITLE
Hosted Launch: Create/Update Deployments

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/2206

We should try to merge this to the base PR before working on the additional Deployment-related docs like for compute queues.

### CODE REVIEW

File: astro/configure-deployment-resources.md [hunk 1]
```diff
@@ -44,22 +44,24 @@ All Deployments use the Celery executor by default. See [Choose an executor](exe
 5. Click **Update**.
 6. Optional. Configure worker queues and tasks to optimize resource usage on your Deployment. See [Configure an executor](executors.md).
 
-## Scheduler resources
+## Scheduler size
 
 The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html) is responsible for monitoring task execution and triggering downstream tasks when the dependencies are met. 
 
-Scheduler resources must be set for each Deployment and are managed separately from cluster-level infrastructure. To ensure that your tasks have the CPU and memory required to complete successfully on Astro, you can set:
+Scheduler resources must be set for each Deployment and are managed separately from cluster-level infrastructure. To ensure that your tasks have the CPU and memory required to complete successfully on Astro, you can provision the scheduler with varying amounts of CPU and memory. See the following table for recommendations of
```
Comment (jwitz) [inline astro/configure-deployment-resources.md:51]: Need a tech reviewer to help with recommendations for scheduler size.

File: astro/configure-deployment-resources.md [hunk 2]
```diff
@@ -117,7 +119,7 @@ Only the users who are members of the target Workspace can access the Deployment
 
 ## Delete a Deployment
 
-When you delete a Deployment, all infrastructure resources assigned to the Deployment are immediately deleted from your data plane. However, the Kubernetes namespace and metadata database for the Deployment are retained for 30 days. Deleted Deployments can't be restored. If you accidentally delete a Deployment, contact [Astronomer support](https://cloud.astronomer.io/support).
+When you delete a Deployment, all infrastructure resources assigned to the Deployment are immediately deleted. However, the Kubernetes namespace and metadata database for the Deployment are retained for 30 days. Deleted Deployments can't be restored. If you accidentally delete a Deployment, contact [Astronomer support](https://cloud.astronomer.io/support).
```
Comment (jwitz) [inline astro/configure-deployment-resources.md:122]: Is this still true?

File: astro/create-deployment.md [hunk 3]
```diff
@@ -23,30 +23,36 @@ Every Deployment is hosted on a single Astro cluster with its own dedicated reso
...
-    - **Worker Type**: Select the worker type for your default worker queue. See [Worker queues](configure-deployment-resources.md#worker-queues).
+    - **Astro Runtime**: Choose which Astro Runtime version you want your Deployment to run. By default, the latest version of Astro Runtime is selected. The Astro Runtime versions available in the Cloud UI are limited to patches for the most recent major and minor releases. Deprecated versions of Astro Runtime aren't available.
+
+        To upgrade the Astro Runtime version for your Deployment, you’ll need to update your Docker image in your Astro project directory. See [Upgrade Astro Runtime](upgrade-runtime.md).
+    
+    - **Scheduler**: Select the amount of resources you want your Deployment scheduler to use. The scheduler is responsible for queueing and scheduling your Airflow tasks. Choose **Small** for most workloads. See [Configure Deployment resources](configure-deployment-resources.md#scheduler-resources) for more information.
```
Comment (jwitz) [inline astro/create-deployment.md:48]:
```suggestion
    - **CI/CD Enforcement**: Select whether you want the Deployment to only accept code deploys from a Workspace API token or Deployment API key.
    - **High availability**: Select whether you want the Deployment to be available across multiple availability zones. 
    - **Scheduler**: Select the amount of resources you want your Deployment scheduler to use. The scheduler is responsible for queueing and scheduling your Airflow tasks. Choose **Small** for most workloads. See [Configure Deployment resources](configure-deployment-resources.md#scheduler-resources) for more information.
```
Is there anything else we can say about high availability? I'm pretty unfamiliar with the feature.

File: astro/create-deployment.md [hunk 4]
```diff
@@ -23,30 +23,36 @@ Every Deployment is hosted on a single Astro cluster with its own dedicated reso
...
+    - **Worker queue**: Configure the `default` worker queue for your Deployment. A worker queue is a group of identically-configured workers responsible for running your tasks. The default options for the `default` worker queue are suitable for most workloads. See [Worker queues](configure-deployment-resources.md#worker-queues).
 
 4. Optional. Edit additional Deployment resource settings. See [Configure Deployment resources](configure-deployment-resources.md). If you don't change any Deployment resource settings, your Deployment is created with the following resources:
 
-    - The celery executor
-    - A worker queue named `default` that runs a maximum of 10 workers. Each of these workers can run a maximum of 16 tasks can run at a time.
-    - A single scheduler with 0.5 CPUs and 1.88 GiB of memory.
+    - The celery executor.
+    - A worker queue named `default` that runs a maximum of 10 machines. Each machine in the queue has 1 vCPU and 2G of RAM, and they can each run a maximum of 5 tasks at a time.
```
Comment (conorbev) [inline astro/create-deployment.md:54]: I'm not sure if I'm missing some context here, but this seems to just describe the "default" that the user will be presented with when they first click "Create Deployment".

The user can choose the worker type they want ( and they can see the CPU/Memory size of each worker ).

They are not limited to 10 workers, they can adjust the min and max as needed. 

The default settings for concurrent tasks are:

A5 - 5 concurrent tasks
A10 - 10 concurrent tasks
A20 - 20 concurrent tasks

^ We don't think a user should adjust those concurrency setting defaults unless they know what they are doing.

It is worth documenting here that all of these worker types come with 5 GiB of ephemeral storage available to them.

I think in terms of "what size worker type do I need?" We can do similar to the Scheduler. Like suggest to start small and then you can observe actual CPU and memory utilization requirements in the deployment analytics tab and adjust from there.

File: astro/create-deployment.md [hunk 5]
```diff
@@ -23,30 +23,36 @@ Every Deployment is hosted on a single Astro cluster with its own dedicated reso
...
+    - A scheduler with 1 CPU and 2 GiB of memory.
```
Comment (conorbev) [inline astro/create-deployment.md:55]: This line should probably be deleted? The scehduler is set elsewhere.

**Review by conorbev — COMMENTED** [review]
Only comment I have here is that it's going to need to be updated for KE/KPO in Hosted with: https://github.com/astronomer/astro/pull/13352

I think this would be a good one for Constance to give a final review on too. ( especially once the KE/KPO stuff is in ).

File: astro/configure-deployment-resources.md [hunk 6]
```diff
@@ -42,28 +42,40 @@ All Deployments use the Celery executor by default. See [Choose an executor](exe
 3. Click **Edit Details**. 
 4. Select **Celery** or **Kubernetes** in the **Executor** list. If you're moving from the Celery to the Kubernetes executor, all existing worker queues are deleted. Running tasks stop gracefully and all new tasks start with the selected executor.
 5. Click **Update**.
-6. Optional. Configure worker queues and tasks to optimize resource usage on your Deployment. See [Configure an executor](executors.md).
+6. Optional. Configure worker queues and tasks to optimize your executor resource usage. See [Configure an executor](executors.md).
```
Comment (paolaperaza) [inline astro/configure-deployment-resources.md:45]:
```suggestion
6. Optional. Configure worker queues and tasks to optimize your infrastructure consumption. See [Configure an executor](executors.md).
```

Something is off here?

File: astro/configure-deployment-resources.md [hunk 7]
```diff
@@ -42,28 +42,40 @@ All Deployments use the Celery executor by default. See [Choose an executor](exe
...
 
-## Scheduler resources
+## Scheduler size
 
 The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html) is responsible for monitoring task execution and triggering downstream tasks when the dependencies are met. 
 
-Scheduler resources must be set for each Deployment and are managed separately from cluster-level infrastructure. To ensure that your tasks have the CPU and memory required to complete successfully on Astro, you can set:
+Scheduler resources must be set for each Deployment and are managed separately from cluster-level infrastructure. To ensure that your tasks have the CPU and memory required to complete successfully on Astro, you can provision the scheduler with varying amounts of CPU and memory.
```
Comment (paolaperaza) [inline astro/configure-deployment-resources.md:51]: We might need to clarify here that 

- scheduler does not autoscale
- scheduled count affects pricing (link to pricing page)

File: astro/configure-deployment-resources.md [hunk 8]
```diff
@@ -42,28 +42,40 @@ All Deployments use the Celery executor by default. See [Choose an executor](exe
...
 3. Click **Edit Details**. 
-4. Edit the scheduler resource settings. See [Scheduler resources](#scheduler-resources).
+4. Choose a scheduler size. See [Scheduler size](#scheduler-size).
 5. Click **Update**.
 
     The Airflow components of your Deployment automatically restart to apply the updated resource allocations. This action is equivalent to deploying code and triggers a rebuild of your Deployment image. If you're using the Celery executor, currently running tasks have 24 hours to complete before their running workers are terminated. See [What happens during a code deploy](deploy-code.md#what-happens-during-a-code-deploy).
 
+:::info Alternative setup for Astro Hybrid
+
+This document applies only to Astro Hybrid users. Astro Hybrid runs is a version of Astro that runs in your own company's cloud. To see whether you're an Astro Hybrid user, open your Organization in the Cloud UI and go to **Settings** > **General**. Your version of Astro is listed under **Product Type**.
```
Comment (paolaperaza) [inline astro/configure-deployment-resources.md:65]:
```suggestion
This document applies only to Astro Hybrid users. Astro Hybrid is a version of Astro that runs in your own company's cloud. To see whether you're an Astro Hybrid user, open your Organization in the Cloud UI and go to **Settings** > **General**. Your version of Astro is listed under **Product Type**.
```

File: astro/configure-deployment-resources.md [hunk 9]
```diff
@@ -99,6 +111,21 @@ By default, Deployments accept code deploys from any authenticated source. When
...
 
+## Enable high availability
+
+:::info
+
+This feature is on by default for Astro Hosted users. 
+
+:::
+
+When a Deployment has high availability enabled, it runs multiple instances of its Airflow components across separate nodes for resiliency. This ensures that your DAGs can continue to run if there's an issue with one of the clusters your component instance is running on.
```
Comment (paolaperaza) [inline astro/configure-deployment-resources.md:122]: Component instance?

Comment (paolaperaza) [inline astro/configure-deployment-resources.md:122]: Prob a liiiitle more detail we can give here.

Which components are HA when this is toggled? Tell folks that affects pricing too

**Review by paolaperaza — APPROVED** [review]
LGTM @jwitz !

File: astro/configure-deployment-resources.md (hunk 6 above)
Comment (jwitz) [reply astro/configure-deployment-resources.md:45]: This sounds like you're consuming infrastructure, I think the OG is alright. But what doesn't gel is that this is still an optional step, I think it makes more sense to have it as a follow up outside of the main process.

File: astro/configure-deployment-resources.md (hunk 7 above)
Comment (jwitz) [reply astro/configure-deployment-resources.md:51]: Do we run more than one scheduler? Note sure what to make of "scheduled count affects pricing", as it seems obvious that more resources = more cost.

**Pull request merged**